.tool-versions

```diff
@@ -1 +1 @@
-ruby latest:2.7
+ruby latest:2.7.6
```

Comment:
Because the `latest:2.7.6` is not a version we can't use in the `.tool_versions`. We can only use it in the CLI when install the version

```suggestion
ruby 2.7.6
```